### PR TITLE
Add SauceLabs web/phone/iphone profile

### PR DESCRIPTION
**Pull request by TatianaTochko**

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -339,6 +339,15 @@ jobs:
...
           done
+          ./gradlew runStories -x testVividusInitialization \
+                               -Pvividus.configuration.environments=system/saucelabs \
+                               -Pvividus.configuration.suite= \
+                               -Pvividus.configuration.profiles=saucelabs/web,web/phone/iphone \
+                               -Pvividus.bdd.story-loader.batch-1.resource-location=story/integration \
+                               -Pvividus.bdd.story-loader.batch-1.resource-include-patterns=ProxyStepsTests.story \
+                               -Pvividus.selenium.grid.username=${SAUCELABS_USER} \
+                               -Pvividus.selenium.grid.password=${SAUCELABS_KEY}
+        done
```
Comment (valfirst) [inline .github/workflows/gradle.yml:350]: invalid usage

File: .github/workflows/gradle.yml [hunk 2]
```diff
@@ -339,6 +339,15 @@ jobs:
                                  -Pvividus.selenium.grid.username=${SAUCELABS_USER} \
                                  -Pvividus.selenium.grid.password=${SAUCELABS_KEY}
           done
+          ./gradlew runStories -x testVividusInitialization \
+                               -Pvividus.configuration.environments=system/saucelabs \
+                               -Pvividus.configuration.suite= \
```
Comment (valfirst) [inline .github/workflows/gradle.yml:344]: use `grid` suite

File: .github/workflows/gradle.yml [hunk 3]
```diff
@@ -339,6 +339,15 @@ jobs:
...
+                               -Pvividus.configuration.profiles=saucelabs/web,web/phone/iphone \
+                               -Pvividus.bdd.story-loader.batch-1.resource-location=story/integration \
+                               -Pvividus.bdd.story-loader.batch-1.resource-include-patterns=ProxyStepsTests.story \
```
Comment (valfirst) [inline .github/workflows/gradle.yml:347]: 2 properties above are not needed

**Review by valfirst — CHANGES_REQUESTED** [review]

File: .github/workflows/gradle.yml [hunk 4]
```diff
@@ -3,15 +3,15 @@ name: Vividus CI
 on:
   push:
     branches:
-    - master
+      - master
```
Comment (valfirst) [inline .github/workflows/gradle.yml:6]: please remove all noisy (whitespace-only) changes

**Review by valfirst — CHANGES_REQUESTED** [review]

**Comment by TatianaTochko** [comment]
Remove changes to https://github.com/vividus-framework/vividus/pull/1023

**Pull request closed without merging**